java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JwtValidatorBuilder.java

```diff
@@ -176,11 +181,27 @@ public JwtValidatorBuilder withValidatorListener(ValidationListener validationLi
 		List<Validator<Token>> allValidators = createDefaultValidators();
 		allValidators.addAll(validators);
 
-		CombiningValidator<Token> combiningValidator = new CombiningValidator<>(allValidators);
+		CombiningValidator<Token> combiningValidator = new CombiningTokenValidator(allValidators);
 		validationListeners.forEach(combiningValidator::registerValidationListener);
 		return combiningValidator;
 	}
 
+	private class CombiningTokenValidator extends CombiningValidator<Token> {
```

Comment:
why is this private class required? there is already a public  CombiningValidator?

Reply:
Moved logging into CombiningValidator<Token>